examples/batch/main.tf

```diff
@@ -30,4 +30,48 @@ resource "azurerm_batch_account" "batch" {
   resource_group_name      = "${azurerm_resource_group.rg.name}"
   location                 = "${azurerm_resource_group.rg.location}"
   storage_account_name     = "${azurerm_storage_account.stor.name}"
+}
+
+resource "azurerm_batch_pool" "fixedpool" {
+  name                   = "myfixedpool"
+  resource_group_name    = "${azurerm_resource_group.rg.name}"
+  account_name 		       = "${azurerm_batch_account.batch.name}"
+  display_name		       = "Fixed Scale Pool"
+  vm_size				         = "Standard_A1"
```

Comment:
Could we run terraform ft on this file to fix the alignment? you can just point it at the folder and it'll find and fmt all terraform files 🙂

Reply:
Done.